yarn-project/stdlib/src/trees/nullifier_leaf.ts

```diff
@@ -104,19 +104,19 @@ export class NullifierLeaf implements IndexedTreeLeaf {
     /**
      * Nullifier value.
      */
-    public nullifier: Fr,
```

Comment:
Had to change it to make TS and C++ serialization names match. We don't own either classes.